<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormEvent</class>
 <widget class="QWidget" name="FormEvent">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>195</width>
    <height>453</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Event Description</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QGroupBox" name="groupBox_Title">
        <property name="title">
         <string>Title</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLineEdit" name="lineEdit_Title"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_Location">
        <property name="title">
         <string>Location</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLineEdit" name="lineEdit_Location"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_Time">
        <property name="title">
         <string>Time</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QCheckBox" name="checkBox_Start">
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>Start</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTimeEdit" name="timeEdit_Start"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QCheckBox" name="checkBox_End">
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>End</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTimeEdit" name="timeEdit_End">
             <property name="enabled">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_TimeZone"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_Class">
        <property name="title">
         <string>Confidence Classification</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QComboBox" name="comboBox_Class">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <item>
            <property name="text">
             <string>PUBLIC (everything visible when shared)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>CONFIDENTIAL (only busy state visible when shared)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>PRIVATE (invisible when shared)</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_Comment">
        <property name="title">
         <string>Comment</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QPlainTextEdit" name="plainTextEdit"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>checkBox_End</sender>
   <signal>clicked(bool)</signal>
   <receiver>timeEdit_End</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>77</x>
     <y>211</y>
    </hint>
    <hint type="destinationlabel">
     <x>178</x>
     <y>211</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_Start</sender>
   <signal>clicked(bool)</signal>
   <receiver>timeEdit_Start</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>77</x>
     <y>183</y>
    </hint>
    <hint type="destinationlabel">
     <x>178</x>
     <y>183</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
